cli_tools/common/disk/inspect.go

```diff
@@ -38,8 +38,18 @@ type Inspector interface {
 
 // InspectionResult contains the partition and boot-related properties of a disk.
 type InspectionResult struct {
-	// HasEFIPartition indicates whether the disk has a EFI partition.
-	HasEFIPartition                    bool
+	// UEFIBootable indicates whether the disk is bootable with UEFI.
+	UEFIBootable bool
+
+	// BIOSBootable indicates whether the disk is bootable with BIOS.
+	// It's only useful for detecting hybrid (UEFI+BIOS) mode. A BIOS-only disk
```

Comment:
My understanding of hybrid is that the disk supports BIOS booting with MBR partitioning, and plus some other GPT booting (usually I've seen GPT+UEFI).

https://www.rodsbooks.com/gdisk/hybrid.html

Do you have a different understanding?